README.md

```diff
@@ -201,6 +213,35 @@ msync update --noop
 msync update -m "Commit message"
 ```
 
+Available parameters for modulesync.yml
+
+* git_base : The default URL to git clone from (Default: 'git@github.com:')
+* namespace : Namespace of the projects to manage (Default: 'puppetlabs')
+* branch : Branch to push to (Default: 'master')
+* branch_prefix : Branch prefix if necessary (ie. HEAD:refs/publish/master/)
```

Comment:
Instead of `branch` and `branch_prefix` could we have `branch` (which would be the local branch) and `remote_branch` (which would default to the value of `branch`)? That way the push looks like `git push origin localbranchname:remotebranchname`, so they could potentially have different names and we don't need to refer to `HEAD:`.